packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -49,7 +48,7 @@ export async function prepareTestEnv(folderName: string, options?: PrepareTestEn
   const cacheFolderPath = path.resolve(globalFolderPath, 'cache');
 
   JSON.parse(readFileSync(path.join(rootFolderPath, 'packages', '@o3r', 'core', 'package.json')).toString());
-  const yarnVersion: string = yarnVersionParam || getYarnVersionFromRoot(rootFolderPath);
+  const yarnVersion: string = yarnVersionParam || (isYarn1() && YARN_1_LATEST_VERSION) || getYarnVersionFromRoot(rootFolderPath);
```

Comment:
`YARN_1_LATEST_VERSION` is always `truthy`

Reply:
the code was removed